### ISSUE TITLE
Wkhtmltopdf error 2 on printing sales order

### ISSUE THREAD

**Issue by mohdshoaib04** [odoo/odoo#26223]
Impacted versions: odoo 11 (sh)

Steps to reproduce: print sales order from print menu

Current behavior: wkhtmltopdf error

Expected behavior: should print pdf

Video/Screenshot link (optional):
Error :

Wkhtmltopdf failed (error code: 2). Message: b'usage: wkhtmltopdf [-h] [--cookie --COOKIE --COOKIE] [--quiet]\n [--page-size --PAGE-SIZE] [--page-width --PAGE-WIDTH]\n [--page-height --PAGE-HEIGHT] [--margin-top --MARGIN-TOP]\n [--margin-left --MARGIN-LEFT]\n [--margin-right --MARGIN-RIGHT]\n [--margin-bottom --MARGIN-BOTTOM]\n [--header-html --HEADER-HTML]\n [--header-spacing --HEADER-SPACING]\n [--header-line --HEADER-LINE] [--footer-html --FOOTER-HTML]\n [--orientation --ORIENTATION] [--dpi --DPI]\n [--viewport-size --VIEWPORT-SIZE] [--version]\n [files [files ...]]\nwkhtmltopdf: error: unrecognized arguments: --disable-local-file-access\n'

**Comment by oerp-odoo** [comment]
Noticed same bug on odoo.sh. I guess odoo.sh is currently outdated and does not have fix, cause after updating code locally, this problem was solved. How long till fix is deployed on odoo.sh? Why you guys are delaying it so much on odoo.sh?:)

**Comment by mohdshoaib04** [comment]
@oerp-odoo Do you know any fixes for this issue ?

**Comment by oerp-odoo** [comment]
I didnt find which particular  commit fixes it, but recently updating odoo
repo locally, solved this issue. Cause i had same issue on my pc which had
older odoo code. Though on odoo.sh we need to wait for odoo, to update
their code..

**Comment by mohdshoaib04** [comment]
Seems, odoo has updated with fix, We are now able to print reports.
@oerp-odoo Thanks for some issue insights.

**Comment by Yenthe666** [comment]
@rim-odoo or @beledouxdenis can one of you check?

**Comment by beledouxdenis** [comment]
This issue has been solved and customers having created a support ticket to help@odoo.com have been notified. I didn't think users would create issues regarding the Odoo.sh platform here so I didn't come to see if there were issues here regarding that matter.

We sincerely apologize for the inconvenience this issue has caused. We learned from today's mistake and will review our platform and internal process so this kind of issue happen less and less.

We hope you will understand.
